Pod/Classes/WPDateTimeHelpers.m

```diff
@@ -1,5 +1,8 @@
 #import "WPDateTimeHelpers.h"
 
+// To use only on unit tests to check results in specific languages
+static NSString *forcedLocaleIdentifier = nil;
```

Comment:
I might be misunderstanding, but why do we need this `static` property? Can't we just have the `setForcedLocaleIdentifier` method set the locale directly on the shared formatter? Are we using this for anything?

Reply:
You are right! I started doing this in another way, then continued to used the variable anyway when switching to this approach. Will fix this.

Reply:
@frosty ready for another look.